br/pkg/lightning/config/config.go

```diff
@@ -1457,6 +1459,7 @@ func NewConfig() *Config {
 			DiskQuota:               ByteSize(math.MaxInt64),
 			DuplicateResolution:     DupeResAlgNone,
 			PausePDSchedulerScope:   PausePDSchedulerScopeTable,
+			BlockSize:               8192,
```

Comment:
In order to keep compatibility, we should use the old effective value 16 * 1024, as in `newSSTWriter` from backend/local/engine.go line 1377